src/date/expand-pattern.js

```diff
@@ -33,10 +55,43 @@ return function( pattern, cldr ) {
 
 	switch ( true ) {
 		case "skeleton" in pattern:
-			result = cldr.main([
-				"dates/calendars/gregorian/dateTimeFormats/availableFormats",
-				pattern.skeleton
-			]);
```

Comment:
@rxaviers sir,

Sir here it is throwing error `E_MISSING_CLDR` error from `cldr.main` itself and not returning `undefined` for skeletons containing both date and time part. Hence we first have to check if skeleton entered has dateSkeleton or timeSkeleton or both before getting result. I am attaching a screenshot that will explain better.
![untitled213](https://cloud.githubusercontent.com/assets/5673050/6841338/ca055b94-d3ae-11e4-91d0-df00a52f2045.png)